package.json

```diff
@@ -15,7 +15,8 @@
     "build": "cross-env BABEL_ENV=default NODE_ENV=production webpack",
     "lint": "eslint .",
     "dev": "cross-env BABEL_ENV=default webpack --watch",
-    "test": "npm run lint && npm run test-unit"
+    "test": "npm run lint && npm run test-unit",
+	"travis-builds": "concurrently \"npm run build\" \"cross-env BABEL_ENV=default webpack\" \"cross-env BABEL_ENV=default NODE_ENV=test webpack\""
```

Comment:
Can't use npm test persee only test env as `test` has the watch flag which will not error properly to fail the CI build.